### PR TITLE
Updated guide with Apache installation steps

### PR DESCRIPTION
**Pull request by amitech**
Added installation for various operating systems

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: client/src/pages/guide/english/apache/index.md [hunk 1]
```diff
@@ -4,5 +4,20 @@ title: Apache
 ## Apache
 
 The Apache HTTP Server, commonly known as Apache, is a free and open-source cross-platform web server, released under the terms of [Apache License 2.0](https://en.wikipedia.org/wiki/Apache_License). Apache is developed and maintained by an open community of developers under the auspices of the Apache Software Foundation.
+Apache runs on 67% of all webservers in the world. It is fast, reliable, and secure. It can be highly customized to meet the needs of many different environments by using extensions and modules
 
 
+### Installation
```
Comment (lynxlynxlynx) [inline client/src/pages/guide/english/apache/index.md]: Like in the other PR, I don't think this section adds any value. Only a subset of platforms and incomplete. Official docs are always better for this.

Comment (QuincyLarson) [reply client/src/pages/guide/english/apache/index.md]: Even though the docs may be more complete, it doesn't hurt to have this section imho.

**Comment by QuincyLarson** [comment]
@amitech Awesome! Thanks! Congrats on your first open source contribution to this repository!

**Comment by amitech** [comment]
Thanks @QuincyLarson will keep improving by adding some improve information.

**Pull request merged**